force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,10 +207,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
+        Integer MILLISECONDS_PER_MINUTE = 60000;
```

Comment:
Whenever someone talks about a constant, it is important to add the final keyword  to it. That means it can never be changed. When you get more advanced in the program you will see constants are normally set at the class level not method. 

	final Integer MILLISECONDS_PER_MINUTE = 60000;